sky/cli.py

```diff
@@ -2972,7 +2972,7 @@ def _list_to_str(lst):
 
     def _output():
         gpu_table = log_utils.create_table(
-            ['NVIDIA_GPU', 'AVAILABLE_QUANTITIES'])
+            ['COMMON_GPU', 'AVAILABLE_QUANTITIES'])
```

Comment:
Nit: also update the comment here?
https://github.com/skypilot-org/skypilot/blob/56970ac22eef245de6ae2871caff61a84dcb3dac/sky/cli.py#L2987

Reply:
Good catch, done.